cilium/cmd/bpf_lb_list.go

```diff
@@ -65,7 +65,7 @@ func dumpSVC(serviceList map[string][]string) {
 		var entry string
 
 		svcKey := key.(lbmap.ServiceKey)
```

Comment:
`svcKey` is used in the few places afterwards. I think it would be safer if you do `svcKey = svcKey.ToHost()` after `svc := svcKey.String()`.